health/micro-ui/web/micro-ui-internals/example/src/UICustomizations.js

```
@@ -805,172 +806,171 @@ export const UICustomizations = {
-                { code: "3", name: "MP_ACTIONS_FREEZE_MICROPLAN" },
-              ];
-            } else {
-              options = [{ code: "1", name: "MP_ACTIONS_VIEW_SUMMARY" }];
+        },
+        config: { enabled: true },
+      });
+
+      switch (key) {
+        case "ACTIONS":
+          // TODO : Replace dummy file id with real file id when API is ready
+          const dummyFile = "c22a7676-d5d7-49b6-bcdb-83e9519f58df"
```

Biome (2.1.2)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Safe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)